src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -321,6 +322,18 @@ namespace AppInstaller::CLI::Workflow
             AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_COMMAND_REQUIRES_ADMIN);
         }
     }
+
+    void EnsureFeatureEnabled::operator()(Execution::Context& context) const
+    {
+        if (!Settings::ExperimentalFeature::IsEnabled(m_feature))
+        {
+            context.Reporter.Error() << Resource::String::FeatureDisabledMessage << ' ' <<
+                Resource::String::FeatureDisabledFeatureName << ' ' <<

```

Comment:
Why not just: ": ".  That is what is happening in the case that the whole command or an argument is not available.